### ISSUE TITLE
Caliban crashes if label "1" doesn't exist in the .trk file lineage.json

### ISSUE THREAD

**Issue by calboi91** [vanvalenlab/caliban#207]
**Describe the bug**
Caliban crashes if label "1" doesn't exist in the .trk file lineage.json

**To Reproduce**
1. Create a .trk file with labels that start with numbers other than 1
2. Run `python3 caliban.py /path/to/trk.trk
3. See error
```
  File "caliban.py", line 5200, in <module>
    review(sys.argv[1])
  File "caliban.py", line 5192, in review
    **load_trk(filename))
  File "caliban.py", line 1945, in __init__
    self.incomplete = {*self.tracks[1]} < TrackReview.possible_keys
KeyError: 1
```

**Expected behavior**
Expect to have the caliban interface load up.

**Desktop (please complete the following information):**
 - OS: Windows 10, using the docker container
 - Python Version is what the container is using (built using the Dockerfile)

**Additional context**
Caliban does load up properly if .trk file lineage.json contains label 1.

**Comment by geneva-miller** [comment]
Yes, currently .trk files must contain label 1 (we also strongly encourage sequential labeling of the objects within the .trk file). @MekWarrior may have additional commentary.

**Comment by calboi91** [comment]
I currently have a dummy label 1 without any info (like frames or parent/daughters) inserted into each .trk file lineage.json. With this, the file loads, and I don't seem to see any issues with the rest of the functions at the moment. Are there any potential issues with having a dummy label without any info, at least when applying annotation functions in Caliban?
